packages/docusaurus-theme-classic/src/theme/DocItem/index.js

```diff
@@ -115,6 +117,11 @@ function DocItem(props) {
                     <span className="badge badge--secondary">
                       Version: {version}
                     </span>
+                    {isOld ? (
+                      <span className="badge badge--secondary margin-horiz--xs">
+                        <Link to="/docs"> Go to latest version </Link>
```

Comment:
The /docs url is configurable, so it's not a good idea to hardcode the link to /docs here, + there's no guarantee that the user used the docs "homePageId" feature.

See https://v2.docusaurus.io/docs/docs-introduction/#docs-only-mode